practice/earthquake/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet"
+    href="http://openlayers.org/en/v3.17.1/css/ol.css" type="text/css">
+    <script src="http://openlayers.org/en/v3.17.1/build/ol.js"
+     type="text/javascript"></script>
```

Comment:
All script tags should go at the end of the `body`. Although the ordering of them there is important based on what dependencies exist.